apps/catalog/catalog/src/app/marketplace/title/avails/avails.component.html

```diff
@@ -6,8 +6,8 @@ <h2>Search & buy Rights</h2>
   </button>
 </header>
 <nav fxLayout fxLayoutAlign="center center" fxLayoutGap="16px">
-  <a mat-flat-button routerLink="map" routerLinkActive="mat-primary" test-id="multiple-territory">Multiple Territory</a>
-  <a mat-flat-button routerLink="calendar" routerLinkActive="mat-primary" test-id="single-territory">Single Territory</a>
+  <a mat-flat-button routerLink="map"  [queryParams]="route.snapshot.queryParams" routerLinkActive="mat-primary" test-id="multiple-territory">Multiple Territory</a>
```

Comment:
```suggestion
  <a mat-flat-button routerLink="map" [queryParams]="route.snapshot.queryParams" routerLinkActive="mat-primary" test-id="multiple-territory">Multiple Territory</a>
```

so passing nested object to `queryParams` works when it comes from `route.snapshot.queryParams` but not if we use `availsForm.value` ?